dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -133,10 +146,298 @@ xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP] ad
 - In a non-cloud environment, by configuring ingressIP range (`ingressIPNetworkCIDR`),
 `service.type=LoadBalancer`, and `service.port.ingressIP`.
 
-The administrator must ensure the external IPs are routed to the nodes and local firewall rules on all nodes allow access to the open port.
+The administrator must ensure the external IPs are routed to the nodes and local
+firewall rules on all nodes allow access to the open port.
+
+Use the same nodes as the router, but advertise them with VRRP and use the DNS
+address of the router nodes.
+
+.Example External IP Configured within a Service Definition
+----
+$ oc create -f - <<INGRESS
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+spec:
+  externalIPs:
+  - 10.9.54.100 <1>
+  ports:
+  - port: 5432
+    protocol: TCP
+  selector:
+    name: postgresql
+  type: LoadBalancer
+INGRESS
+----
+<1> The IP address assigned to the external IP.
+
+The external IP address is not managed by the underlying Kubernetes
+infrastructure and must be maintained and provided by a cluster administrator.
+While external IPs provide a solution for accessing services on the {product-tlte}
+cluster, there are several shortcomings:
+
+* Application users cannot create external IP addresses themselves, and must work
+with cluster administrators to allocate an address for their usage.
+* There are no protections in place to restrict the usage of a particular external
+address configured within the cluster. This allows the potential for a single
+address to be used by multiple services targeting the same port. When this
+situation occurs, the service which requested the port first is given use of the
+port.
+
+xref:using-ingress[Ingress] provides a solution.
+
+[[using-ingress]]
+=== Ingress
```

Comment:
We can not call it this.  Ingress is too generic and the term is already used in kubernetes for something else.

I have no great ideas for another name, just some bad ones:
- External IP Self-Service
- External IP Auto-allocation
- Ingress IP Self-Service or Auto-Allocation